### DISCUSSION TITLE
Does nicegui have static cache for its own files?

### DISCUSSION THREAD

**Discussion by Wzhipeng** [zauberzeug/nicegui#1785] [Q&A]
I run the program on the server through ui.run, and then access the page in the browser. I find that many js and css files are loaded. Is there any parameter configuration to cache all these static files?

**Comment by falkoschindler** [comment]
If you access a website which is hosted on a server, your browser will usually cache static files (unless you deactivate this feature). So there's not much for NiceGUI to cache. But maybe I misunderstood your scenario.
